skype/skype-ps/skype/New-CsHuntGroup.md

```diff
@@ -449,6 +487,8 @@ This cmdlet supports the common parameters: `-Debug, -ErrorAction, -ErrorVariabl
 
 ## OUTPUTS
 
+### Microsoft.Rtc.Management.Hosted.HuntGroup.NewCsHuntGroupCmdlet
```

Comment:
Should be:
Microsoft.Rtc.Management.Hosted.HuntGroup.Models.HuntGroup

Reply:
Fixed